packages/terra-theme-context/src/terra-dev-site/doc/ThemeContext/ThemeContext.1.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import { Badge } from 'terra-theme-context/package.json?dev-site-package';
+
+import ThemeContextProviderExample from '../example/ThemeContextProviderExample?dev-site-example';
+import ThemedComponent from '../example/ThemedComponent?dev-site-example';
+
+<Badge />
+
+# Terra Theme Context
+A react context to provide the current theme. This package is intended to be used by terra components only, Any terra consumers or custom terra components should consume the context from the terra-application package.
```

Comment:
```suggestion
A React context to provide the current theme. This package is intended to be used by Terra components only. Any terra consumers or custom Terra components should consume the context from the `terra-application` package.
```

Reply:
> Any terra consumers or custom Terra components should consume the context from the `terra-application` package.

You said we didn't want to add this directly to terra-application due to cyclical dependencies. Doesn't this recommendation create that cyclical dependency?

Reply:
No, Consumers of terra use the dependency from terra-application. Our terra-components use the context directly.